Makefile

```
@@ -417,12 +417,14 @@ RESET := $(shell tput sgr0 2>/dev/null)
 fmt: fmt/ts fmt/go fmt/terraform fmt/shfmt fmt/prettier
 .PHONY: fmt
 
+GO_FMT_FILES := $(shell find . $(FIND_EXCLUSIONS) -type f -name '*.go' -exec sh -c 'grep -L "DO NOT EDIT" "{}" 2>/dev/null || true' \;)
```

```suggestion
GO_FMT_FILES := $(shell find . $(FIND_EXCLUSIONS) -type f -name '*.go' -exec sh -c 'grep -L "DO NOT EDIT" {} 2>/dev/null || true' \;)
```
`{}` is a shell-safe replacement, not supposed to be quoted:

```
❯ touch 'foo"bar"'
❯ find . -type f -exec sh -c 'grep foo "{}"' \;
grep: ./foobar: No such file or directory
```

---

Actually, calling `sh` here is the main problem, but quoting `{}` should also be avoided 😄.